### PR TITLE
normal local Signup implementation

### PR DESCRIPTION
**Pull request by H4kim**
normal local Signup implementation

### CODE REVIEW

**Review by aorelhas — CHANGES_REQUESTED** [review]
Inside authController, we must validate if the users already exists in the db and if it already exists, we should handle this 'error'. 
And we can use express-validator and check if the principal fields are not empty.

**Comment by H4kim** [comment]
I didn't handle errors  because i want to add error handler middleware , i want to keep things separate and handle error in the next PR

File: models/user.js [hunk 1]
```diff
@@ -1,17 +1,46 @@
-const mongoose = require('mongoose');
-const {Monitor} = require('./monitor');
+const mongoose = require("mongoose");
+const { Monitor } = require("./monitor");
+const bcrypt = require("bcrypt");
 const Schema = mongoose.Schema;
 
 const User = new Schema({
-    username: { type: String , unique: true },
```
Comment (priyajitm) [inline models/user.js:6]: I would suggest against using unique in mongoose schema, I have seen this to not work sometime depending on the data processing. I would not take a risk with this.

**Comment by H4kim** [comment]
closed

**Pull request closed without merging**